RELEASES.md

```diff
@@ -1,5 +1,27 @@
 # Releases
 
+## 2.5.11 (January 15th, 2021)
+
+### Changelog
+
+- #1452 [BUG] v2.5.10 Snippet Initialization fails to redirect proxied functions -- causing terminal exception
+- #1434 Adding cookie path to make AI client SDK work behind Azure Application Gateway
+
+### Update React plugin to v3.0.6
```

Comment:
Just a general nit, i've been not adding these to the releases.md until the actual release pr for bumping the numbers up, this is because we need to publish 2.5.11 before we can update the package.json of these....

And technically, this time around, i'm not seeing anything in 2.5.11 that actually requires these to be released. -- so we may not. Especially as we have Xiao's PR for Angular.

Reply:
Thanks for clarifying, reverting these changes